kubejs/server_scripts/expert/recipes/enigmatica/dust_crushing_tiers.js

```diff
@@ -46,6 +46,8 @@ ServerEvents.recipes((event) => {
 
     const metals = Object.keys(metal_properties);
 
+    // Note to Motoko
+    // Due to issues with AU and hard-coded content, vanilla ingots are now from Minecraft! This script requires a bit of a rework to add support for minecraft namespace -- Kanz
```

Comment:
What is this about? Which issues and what hard-coded content?

Reply:
Anvil repairs mostly.

Reply:
Yea, non-JSON-based recipes can't be covered by AU. The other issues, like smithing-like recipes, should be resolved. If you have further issues, feel free to talk to us.